nbs/tutorial_colab.ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>This picture didn't show up in the rendered docs.  You must use nbdev_detach or something similar.  Also make sure you are previewing the docs locally first.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/398/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
All images are hosted on imgur, not locally, so this shouldn't be an issue